src/plugins/NativeBasePlugin.ts

```diff
@@ -0,0 +1,25 @@
+import Spin from "../Spin";
+import { ConfigPlugin } from "../ConfigPlugin";
+import { Builder } from "../Builder";
+import findJSRule from './shared/JSRuleFinder';
+
+export default class NativeBasePlugin implements ConfigPlugin {
+    configure(builder: Builder, spin: Spin) {
+        const stack = builder.stack;
+
+        if (stack.hasAll(['native-base', 'react-native', 'webpack'])) {
+
+            const jsRule = findJSRule(builder);
+            const combinedJsRule = spin.merge(jsRule, {
+                exclude: /node_modules\/(?!native-base)$/,
```

Comment:
I'm not sure whether this value is correct, as this value is set by `react-native` plugin here:
https://github.com/sysgears/spin.js/blob/master/src/plugins/ReactNativePlugin.ts#L54
so you need all the value from `react-native` plugin plus part for `native-base`

Reply:
I wasn't quite sure how to merge them together without a straight copy, any ideas?

Reply:
I think it will work because of this: https://github.com/survivejs/webpack-merge/blob/90cfcb29b2815b4831eee5468a7dea6bc78e12d3/test.js#L465

Reply:
@justinr1234 Maybe, just expect that react-native plugin will set something before NativeBasePlugin and that the format of this RegEx will be the same as now. So just add |native-base before )$ in RegEx. Something like this. I think no need to be too smart at this phase

Reply:
@justinr1234 If it will just work as is - it is great!

Reply:
@vlasenko I think it might work as is because it should combine them into an array of excludes based on that `webpack-merge` test.

Reply:
Running locally I see no errors and the `packagerOpts` got properly generated too. I'll get some console logging going to verify.

Reply:
Cool!

Reply:
It comes out like this, do you see any issue with having 2 blocks like this?

```
   { rules: 
      [ { test: /\.jsx?$/,
          exclude: /node_modules\/(?!react-native|@expo|expo|lottie-react-native|haul|pretty-format|react-navigation)$/,
          use: [Function] },
        { test: /\.graphqls/, use: 'raw-loader' },
        { test: /\.(graphql|gql)$/,
          exclude: /node_modules/,
          use: [ 'graphql-tag/loader' ] },
        { test: /\.(bmp|gif|jpg|jpeg|png|psd|svg|webp|m4v|aac|aiff|caf|m4a|mp3|wav|html|pdf|ttf)$/,
          use: 
           { loader: '/Users/justin/bv/ironworks_2/spin.js/lib/plugins/react-native/assetLoader.js',
             query: 
              { platform: 'ios',
                root: '/Users/justin/bv/ironworks_2/iron-works',
                bundle: false } } },
        { test: /\.jsx?$/,
          exclude: /node_modules\/(?!native-base)$/,
          use: [Function] } ] },
  resolve: 
```

Reply:
@justinr1234 I'm not sure how Webpack will handle this. I think there should be only one rule for .js in config

Reply:
@justinr1234 Maybe just do jsRule.exclude = []/node_modules\/(?!native-base)$/.concat(jsRule.exclude) instead of using webpack-merge for this?

Reply:
That worked